tic-tac-toe-app/components/Game.jsx

```diff
@@ -0,0 +1,145 @@
+function Board() {
+    const [squares, setSquares] = React.useState(new Array(9).fill(null));
+
+    squares[index] = props.turn;
+    setSquares(squares[index]);
+
+    const declareWinner = () => {
+        const conditions = [
+            [0, 1, 2],
+            [3, 4, 5],
+            [6, 7, 8],
+            [0, 3, 6],
+            [1, 4, 7],
+            [2, 5, 8],
+            [0, 4, 8],
+            [2, 4, 6]
+        ];
+
+        for (let i = 0; i < conditions.length; i++) {
+            const [first, second, third] = conditions[i];
+            if (squares[first] && squares[second] === squares[first] && squares[third] === squares[first]) {
+                return squares[first];
+            }
+        }
+    }
+
+    const winner = declareWinner(squares);
+    
+    let status;
+    if (winner) {
+        status = `The winner is ${winner}`;
+    }
+
+    function clickHandle(index) {
+        if (!winner || !squares[index]) {
+            squares[index] = props.turn;
+            setSquares(squares)
+            props.setTurn(() => {
+                if (props.turn === 'X') {
+                    props.turn = 'O'
+                }
+                props.turn = 'X'
+            })
+        }
+    }
+
+    return (
+            <section className='board'>
+                <h3>The winner is {winner}</h3>
+                <section className="row row1">
+                    <Square onClick={() => pickSquare(0)} value={setSquares[0]}/>
+                    <Square onClick={() => pickSquare(1)} value={setSquares[1]}/>
+                    <Square onClick={() => pickSquare(2)} value={setSquares[2]}/>
+                </section>
+                <section className="row row2">
+                    <Square onClick={() => pickSquare(3)} value={setSquares[3]}/>
+                    <Square onClick={() => pickSquare(4)} value={setSquares[4]}/>
+                    <Square onClick={() => pickSquare(5)} value={setSquares[5]}/>
+                </section>
+                <section className="row row3">
+                    <Square onClick={() => pickSquare(6)} value={setSquares[6]}/>
+                    <Square onClick={() => pickSquare(7)} value={setSquares[7]}/>
+                    <Square onClick={() => pickSquare(8)} value={setSquares[8]}/>
+                </section>
+            </section>
+        )
+    }
+
+function Game() {
+
+    function Square(props) {
+        return <button className='square' onClick={props.click}>{props.value}</button>
+    }
+
+    function Board(props) {
```

Comment:
There is another Board declared in the outer scope on line 1. What is the relationship meant to be between the two components?